cmd/internal/root.go

```diff
@@ -129,6 +138,7 @@ func init() {
 	pf.Int("max-extendedjob-workers", 1, "Maximum of number concurrently running ExtendedJob controller")
 	pf.Int("max-extendedsecret-workers", 5, "Maximum of number concurrently running ExtendedSecret controller")
 	pf.Int("max-extendedstatefulset-workers", 1, "Maximum of number concurrently running ExtendedStatefulSet controller")
+	pf.Bool("apply-crd", true, "If true, apply CRDs on start")
```

Comment:
false so we are backwards compatible for now

Reply:
done